source/ed/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ SET(SOURCE_LIB
 )
 
 add_library(ed ${SOURCE_LIB})
-target_link_libraries(ed types fare)
+target_link_libraries(ed types fare data)
```

Comment:
why `ed` is also involved?

Reply:
what happens is that I store the protobuf Address in the StopPoint object (maybe not the best thing to do), consequently `types` depends on the protobuf files, and this impact dependencies everywhere :crying_cat_face: